plugin/pkg/auth/authorizer/node/graph.go

```diff
@@ -288,3 +329,41 @@ func (g *Graph) DeleteVolumeAttachment(name string) {
 	defer g.lock.Unlock()
 	g.deleteVertex_locked(vaVertexType, "", name)
 }
+
+// SetNodeConfigMap sets up edges for the Node.Spec.ConfigSource.ConfigMapRef relationship:
+//
+// configmap -> node
+func (g *Graph) SetNodeConfigMap(nodeName, configMapName, configMapNamespace string) {
+	g.lock.Lock()
+	defer g.lock.Unlock()
+
+	// nothing to do if we don't have a node
+	if len(nodeName) > 0 {
```

Comment:
calling this with an empty node name is an error... shouldn't need to check this